opensrp-reveal/src/test/java/org/smartregister/reveal/interactor/ListTaskInteractorTest.java

```diff
@@ -197,11 +213,71 @@
             "  }\n" +
             "}";
 
+    @Captor
+    private ArgumentCaptor<JSONObject> captor;
+
+
+    @Before
+    public void setUp() {
+        listTaskInteractor = new ListTaskInteractor(new AppExecutors());
+    }
+
     @Test
     public void testSaveJsonFormShouldSaveMosquitoCollectionForm() throws Exception {
-        ListTaskInteractor listTaskInteractorSpy = spy(listTaskInteractor);
+        ListTaskInteractor listTaskInteractorSpy = PowerMockito.spy(listTaskInteractor);
+
         doNothing().when(listTaskInteractorSpy, "saveMosquitoCollectionForm", any(JSONObject.class));
```

Comment:
Remove this, the object being unit tested should not be disabled from executing methods.

Only the dependencies should be mocked or disabled from executing methods but not the object being unit tested

To fully test this method test test that below were called
 `eventClientRepository.addEvent` 
`clientProcessor.calculateBusinessStatus`
`presenterCallBack.onFormSaved`

This should remain only you test the method ``

Reply:
`ListTaskInteractor` stubbing removed.